src/argilla/client/feedback/training/schemas.py

```diff
@@ -724,8 +780,8 @@ class TrainingTaskForSFT(BaseModel, TrainingData):
         >>> def formatting_func(sample: Dict[str, Any]):
         ...     annotations = sample["good]
         ...     if annotations and annotations[0]["value"] == "Bad":
-        ...         return
-        ...     return template.format(prompt=sample["prompt"][0]["value"], response=sample["response"][0]["value"])
+        ...         return None
```

Comment:
None doesn't need to be specified, just `return` works

Reply:
idem